src/main/java/fr/xephi/authme/command/executable/email/ShowEmailCommand.java

```diff
@@ -17,6 +19,9 @@
 public class ShowEmailCommand extends PlayerCommand {
 
     @Inject
+    private Settings settings;
```

Comment:
You can use `CommonService#getProperty` ;) `CommonService` just offers the most used stuff from `Settings`, `Messages` and `PermissionsManager` to reduce the number of injected services.